### PR TITLE
Added item component

### PR DESCRIPTION
**Pull request by makarman**
Added Item compomemt
Added Assignment subcomponent
Added Literature subcomponent
Added Answer subcomponent
Added Timeleft subcomponent

## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVPYTP-51)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/containers/item/answer.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
Comment (redmyt) [inline eventually/static/src/containers/item/answer.js]: name of files with components should start with the uppercase letter.

File: eventually/static/src/containers/item/answer.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import {withRouter} from 'react-router-dom'
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import {sendAnswer} from './itemService.js'
+
+const style = {
```
Comment (redmyt) [inline eventually/static/src/containers/item/answer.js]: please, use 4 spaces indentation in your files =)

File: eventually/static/src/containers/item/answer.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        <br />
+        <RaisedButton label='Send Answer'
+                      primary={true}
+                      onClick={this.handleAnswer}/>
+      </div>
+    );
+  };
+};
+
+export default withRouter(Answer)
```
Comment (redmyt) [inline eventually/static/src/containers/item/answer.js]: is there necessary to use `withRouter` export?
Add new line =)

File: eventually/static/src/containers/item/assignment.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    return(
+      <div style={style} >
+        <h2>Assignment</h2>
+        <p>{this.props.description}</p>
+      </div>
+    );
+  };
+};
+
+export default Assignment
```
Comment (redmyt) [inline eventually/static/src/containers/item/assignment.js]: new line

File: eventually/static/src/containers/item/item.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    super(props);
+    this.state = {
+      description: 'It is your first task and you have to read books from literature list',
+      literatureList: ['Learn Python in 3 days', 'Think Python', 'Dive into Python' ],
+      timeToLive: 60*60*1000,
+    }
+  }
+
+  render() {
+      console.log(this.state['literatureList'])
```
Comment (redmyt) [inline eventually/static/src/containers/item/item.js]: forgot it =)

File: eventually/static/src/containers/item/literature.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+              {item}
+            </li>
+          ))}
+        </ul>
+      </div>
+    );
+  };
+};
+
+export default withRouter(Literature)
```
Comment (redmyt) [inline eventually/static/src/containers/item/literature.js]: new line

File: eventually/static/src/containers/item/itemService.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import axios from "axios"
+
+const appPath = '/api/v1/user/'
```
Comment (redmyt) [inline eventually/static/src/containers/item/itemService.js]: are all of this path necessary?

File: eventually/static/src/containers/item/answer.js (hunk 1 above)
Comment (skramar) [reply eventually/static/src/containers/item/answer.js]: Components start with Uppercase, files camelCase

File: eventually/static/src/containers/home.js [hunk 8]
```diff
@@ -7,7 +7,7 @@ export default class Home extends React.Component {
       return (
         <div>
           <h1>Hello user!!!!</h1>
-
+
```
Comment (lhalam) [inline eventually/static/src/containers/home.js]: remove empty line

File: eventually/static/src/containers/item/answer.js (hunk 1 above)
Comment (redmyt) [reply eventually/static/src/containers/item/answer.js]: I found that advice in following articles: (https://daveceddia.com/react-project-structure/, https://survivejs.com/react/advanced-techniques/structuring-react-projects/) and also WebStror IDE notifies me about it =)

File: eventually/static/src/containers/home.js (hunk 8 above)
Comment (makarman) [reply eventually/static/src/containers/home.js]: done

File: eventually/static/src/containers/item/answer.js (hunk 1 above)
Comment (makarman) [reply eventually/static/src/containers/item/answer.js]: Okey I'll use uppercase

File: eventually/static/src/containers/item/answer.js (hunk 2 above)
Comment (makarman) [reply eventually/static/src/containers/item/answer.js]: done

File: eventually/static/src/containers/item/answer.js (hunk 3 above)
Comment (makarman) [reply eventually/static/src/containers/item/answer.js]: done

File: eventually/static/src/containers/item/assignment.js (hunk 4 above)
Comment (makarman) [reply eventually/static/src/containers/item/assignment.js]: done

File: eventually/static/src/containers/item/item.js (hunk 5 above)
Comment (makarman) [reply eventually/static/src/containers/item/item.js]: done

File: eventually/static/src/containers/item/itemService.js (hunk 7 above)
Comment (makarman) [reply eventually/static/src/containers/item/itemService.js]: no) I'll remove it

File: eventually/static/src/containers/item/literature.js (hunk 6 above)
Comment (makarman) [reply eventually/static/src/containers/item/literature.js]: done

**Review by lhalam — DISMISSED** [review]

**Review by lhalam — DISMISSED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**